taxi/forms.py

```diff
@@ -0,0 +1,53 @@
+import re
+
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from .models import Car, Driver
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data.get("license_number")
+        if not re.match(r"^[A-Z]{3}\d{5}$", license_number):
+            raise forms.ValidationError(
+                "License must consist of 3 uppercase "
+                "letters followed by 5 digits."
+            )
+        return license_number
+
+
+class DriverCreationForm(UserCreationForm):
+
+    class Meta(UserCreationForm.Meta):
```

Comment:
this form must validate license as well - use inheritance feature to implement this